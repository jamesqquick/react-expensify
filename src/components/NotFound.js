import React from 'react';
import { NavLink } from 'react-router-dom';
const NotFoundPageComponent = () => (
	<div>
		404!
		<NavLink to="/">Go Home</NavLink>
	</div>
);

export default NotFoundPageComponent;
