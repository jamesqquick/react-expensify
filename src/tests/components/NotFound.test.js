import React from 'react';
import NotFound from '../../components/NotFound';
import { shallow } from 'enzyme';

test('should render NotFound correctly', () => {
	const wrapper = new shallow(<NotFound />);
	expect(wrapper).toMatchSnapshot();
});
