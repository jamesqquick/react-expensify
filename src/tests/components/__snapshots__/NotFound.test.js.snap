// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<div>
  404!
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/"
  >
    Go Home
  </NavLink>
</div>
`;
